examples/simple_nve/lammps_input.script

```diff
@@ -0,0 +1,46 @@
+# LAMMPS Simple MD for validation of SimPhoNy lammps-md wrapper. 
+# 
+
+dimension 3
+boundary p p p
+
+atom_style  atomic
+neighbor    0.3 bin
+neigh_modify    delay 5
+
+pair_style lj/cut 2.5
+# read from SimPhoNy-generated initial configuration
+
+read_data data_input_from_sim.lammps 
```

Comment:
I think we should use `create_atoms' command to generate the atoms.  Then we can remove data_input_from_sim.lammps